hardware/diver/main/Core/Inc/banks/BankLayout.h

```diff
@@ -1,5 +1,8 @@
 #include "WavePlusLUT.h"
 
+#include "VanTaBanks.h"
```

Comment:
I'm not sold on the "separate files by author" strategy yet. Can we table that discussion for now and just add your new stuff directly to BankLayout.h? Otherwise, looks great!